tools/ci_build/github/apple/coreml_supported_ops.md

```diff
@@ -17,7 +17,7 @@ Keep in sync with doco generated from /docs/execution-providers/CoreML-Execution
 |ai.onnx:GlobalMaxPool|Only 2D Pool is supported.|
 |ai.onnx:MatMul|Input B should be constant.|
 |ai.onnx:MaxPool|Only 2D Pool is supported.|
-|ai.onnx:PRelu|Input slope should be constant.<br/>Input slope should either have shape [C, 1, 1] or have 1 element.|
+|ai.onnx:PRelu|Input derivative should be constant.<br/>Input derivative should either have shape [C, 1, 1] or have 1 element.|
```

Comment:
could we just mark this one as a false positive?
the ONNX spec also uses "slope", it might be confusing to refer to it by another name.
https://github.com/onnx/onnx/blob/main/docs/Operators.md#PRelu

Reply:
Either works. Here is the exclusion file: https://github.com/microsoft/onnxruntime/blob/main/tools/ci_build/policheck_exclusions.xml